src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Refactoring/RenameEndpointTest.cs

```diff
@@ -378,6 +380,51 @@ public async Task Handle_Rename_DifferentDirectories()
                 });
         }
 
+        [Fact]
+        public async Task Handle_Rename_SingleServerRename_CallsDelegatedLanguageServer()
```

Comment:
> I still need to write a test for the custom message target stuff, ideally that calls the actual C# language server, I just haven't yet.

Ahh I see. Ya definitely more work. were you thinking of doing that in a follow up? If you're curious [here's](https://github.com/dotnet/razor-tooling/blob/e95f3afeaca8c408744bd59e2a849fbbf63fd714/src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/Delegation/DelegatedCompletionListProviderTest.cs#L134) how I was able to get completion to use the real server

Reply:
I'll do it here, just put this up to see if there were major objections